service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -525,7 +525,9 @@ private ApiCreateCloudContextResult fetchCreateCloudContextResult(
                     gcpCloudContextService.getRequiredGcpProject(workspaceId), userRequest),
             "enablePet");
     UserWithPetSa userAndPet = new UserWithPetSa(userEmail, petSaEmail);
-    petSaService.enablePetServiceAccountImpersonation(workspaceId, userAndPet);
+    String proxyGroupEmail =
```

Comment:
Should we put these two lines into PetSaService?  If it is used elsewhere, it might be worth it.

Reply:
Done